sdk/core/azure-core/src/main/java/com/azure/core/util/TransferUtil.java

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util;
```

Comment:
Does anywhere outside of Core call into this? If not, let's put this in implementation until that is needed.

Reply:
yes, storage is going to use it.